### PR TITLE
Coverage increased

### PR DESCRIPTION
**Pull request by pioardi**
## PR Checklist

- [x] Please add a description of your changes.
- [x] We need your changes to come with unit tests in order to keep this project in quality and easy to maintain.
- [ ] Please add a link to the open issue or task that this pull request will resolve.
- [ ] Add a note to the changelog to indicate the change.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

If yes please indicate it and migration info into the CHANGELOG.md file.

---

Increase coverage adding more unit tests

### CODE REVIEW

**Comment by pioardi** [comment]
@jerome-benoit @Shinigami92 I would let you add unit tests for strategies if you have time today into this PR or another.
After this other unit tests we only need to document selection strategies and we can then release the stable 2.0.0 version

**Comment by Shinigami92** [comment]
> @jerome-benoit @Shinigami92 I would let you add unit tests for strategies if you have time today into this PR or another.
> After this other unit tests we only need to document selection strategies and we can then release the stable 2.0.0 version

I don't know if I will have time before ~15:00
And even then I will maybe rest a bit after work in some games
So currently maybe only in review-mode

**Comment by pioardi** [comment]
Nevermind I will let you go though that when you feel is needed.
The only point where the coverage is missing on the selection strategies are the default values.

File: tests/worker/abstract-worker.test.js [hunk 1]
```diff
@@ -1,10 +1,28 @@
...
+
+  it('Verify that handle Error function is working properly', () => {
+    const error = new Error('My error')
+    const worker = new ThreadWorker(() => {})
+    expect(worker.handleError(error)).toBe(error)
+  })
+
+  it('Verify that get main worker throw error if main worker is not set', () => {
+    const worker = new StubPoolWithIsMainWorker(() => {})
+    expect(() => worker.getMainWorker()).toThrowError()
```
Comment (jerome-benoit) [inline tests/worker/abstract-worker.test.js:26]: You can remove the useless intermediate worker var: 

expect(() => new StubPoolWithIsMainWorker(() => {}).getMainWorker()).toThrowError()

**Review by jerome-benoit — CHANGES_REQUESTED** [review]

File: tests/worker/abstract-worker.test.js (hunk 1 above)
Comment (pioardi) [reply tests/worker/abstract-worker.test.js:26]: I see you are starting to appreciate sonar :P

Comment (pioardi) [reply tests/worker/abstract-worker.test.js]: Done

**Review by jerome-benoit — APPROVED** [review]

**Pull request merged**